docs/pages/use-teleport/teleport-connect.mdx

```diff
@@ -115,3 +115,20 @@ menu item.
 Teleport Connect updates and caches the local shell environment on app restart and not when starting
 a new shell session. If you add new environment variables to your shell startup files, Connect will
 see them only after you restart the app.
+
+### Insecure mode
+
+The insecure mode skips the verification of the HTTPS certificate when talking to a proxy service of
```

Comment:
```suggestion
You can open Teleport Connect in insecure mode, which skips HTTPS certificate verification when talking to a Teleport Proxy Service.
```
Suggested a way to tighten this up a bit. Also capitalizing Proxy Service since it's a product name.

Reply:
Yeah, we can go with Teleport Proxy Service. I used "a proxy service of a Teleport Cluster" because I wasn't sure if a regular end user of Teleport will know that "proxy service" means the thing Connect hits after you provide the URL for a cluster.